devtools/Makefile

```diff
@@ -25,3 +25,23 @@ uninstall:
 	rm ${TOPDIR}/devtools/bin/td-layout-config
 	rm ${TOPDIR}/devtools/bin/test-runner-server
 
+tools-init:
+	podman build -t tools dev_container
```

Comment:
`podman` may only work on Red Hat based distribution, so should we also support docker too?

Reply:
Done